### PR TITLE
Change ListRepos and ListUserRepos to return new ListRepositories struct with count

### LINKED ISSUE #1791: Return TotalCount in ListRepos in AppsService
`ListRepos` currently only returns the list of repositories from github's response but not the total_count ([Link](https://docs.github.com/en/rest/reference/apps#list-repositories-accessible-to-the-app-installation)). This total_count is different from just the length of the list of repositories returned if the number of repositories in the installation is greater than the per_page pagination limit set in the request. Is it possible to add it for ListRepos to return?

### PR DESCRIPTION
**Pull request by DarthBenro008**
- Fixes #1791
- Modified Tests in `apps_installations_test.go`

### CODE REVIEW

File: github/apps_installation.go
Comment (gmlewis) [inline github/apps_installation.go:13]: Can you please add a godoc-style comment here?  Maybe something like this?

```
// ListRespositories represents the response from the list repos endpoints.
```

**Review by gmlewis — CHANGES_REQUESTED** [review]
Thank you, @DarthBenro008 !
Just one minor comment addition, please, then we should be ready for a second LGTM before merging.

File: github/apps_installation.go
Comment (DarthBenro008) [reply github/apps_installation.go:14]: Added the godoc-style comment!

**Review by gmlewis — APPROVED** [review]
Thank you, @DarthBenro008 !
LGTM.

Awaiting second LGTM before merging.

**Review by wesleimp — APPROVED** [review]
LGTM 👌🏼

**Comment by gmlewis** [comment]
Thank you, @wesleimp !
Merging.

**Pull request merged**
